src/common/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,20 +1,21 @@
 import { Tooltip, Typography } from "antd";
-import React, { ReactElement } from "react";
+import React, { ForwardedRef, ReactElement } from "react";
 import { useSelector } from "react-redux";
 import { selectCurrentPage } from "../../../app/main.selectors";
 import { PageType } from "../../../app/types/mainSlice.types";
 import { selectCurrentPageObject } from "../../../features/pageObjects/pageObject.selectors";
 
-export const Breadcrumbs = (): ReactElement | null => {
+// eslint-disable-next-line react/display-name
```

Comment:
no need to add this comment. Just add `Breadcrumbs.displayName = "Breadcrumbs";` after the function